tests/preprocessing/test_data.py

```diff
@@ -561,11 +563,12 @@ def test_df_transform(self, daskify):
         res_df = a.fit_transform(frame)
         res_arr = b.fit_transform(frame)
         res_c = c.fit_transform(frame)
+
         if daskify:
             res_pandas = a.fit_transform(frame.compute())
             assert dask.is_dask_collection(res_df)
             assert dask.is_dask_collection(res_arr)
-            assert_eq_df(res_df.compute().reset_index(drop=True), res_pandas)
+            assert_eq_df(res_df.compute(), res_pandas)
```

Comment:
You can skip the compute here. `assert_eq` will handle it for you.

(though I see now that this had `compute` before).